### DISCUSSION TITLE
SideStick

### DISCUSSION THREAD

**Discussion by Flight2117** [flybywiresim/aircraft#4715] [Help] [answered]
Hi, my sidestick isn't moving left or right. Can only move up & down. I've reinstalled it many times but still the same issue.

**Comment by ExampleWasTaken** [comment, marked as answer]
Make sure to use an axis and not 0-100%.

**Reply by Flight2117** [reply]
How does that work? and by the way i still use my keyboard for the controls. I don't have a gear yet. 

On the stable version it works fine but on the dev. en exp. version it doesnt work. :(

**Reply by aguther** [reply]
https://docs.flybywiresim.com/start/autopilot-fbw/

Known issue due to a problem in the SDK.
